### PR TITLE
Debug fixes

### PR DESCRIPTION
**Pull request by qpresley**
Updating some functions with debug statements and removing some commented out code.

@Amir-61 PTAL

### CODE REVIEW

File: lib/ibmdb.js [hunk 1]
```diff
@@ -757,18 +765,11 @@ IBMDB.prototype.buildIndexes = function(model) {
...
-  //   if (propIndex) {
-  //     indexClauses.push(propIndex);
-  //   }
-  // }
-
   return indexClauses;
 };
 
 IBMDB.prototype.buildColumnDefinition = function(model, prop) {
+  debug('IBMDB.prototype.buildColumnDefinition');
```
Comment (Amir-61) [inline lib/ibmdb.js]: Any reason why not adding some arguments in `debug`?

**Comment by Amir-61** [comment]
@qpresley You may want to backport it to `loopback-2.x` as well.

File: lib/ibmdb.js (hunk 1 above)
Comment (qpresley) [inline lib/ibmdb.js]: Fixing now.  On your other comments, I haven't back fit to 2.x yet so will need some assistance with that.

Comment (Amir-61) [inline lib/ibmdb.js]: @qpresley 
>I haven't back fit to 2.x yet so will need some assistance with that.

That's how I would do it:
```
# Go to 2.x branch
git checkout loopback-2.x
# create a feature branch
git checkout -b backport/add-debig-2.x
git cherry-pick <sha>
git push origin backport/add-debig-2.x -u
```
You can get the `sha` key of you commit after landing this PR and going to https://github.com/strongloop/loopback-ibmdb/commits/master, then you would be able to copy the `sha` key of the commit you want to backport; please see:

screen shot 2016-09-28 at 12 24 37 pm


I hope thats clear... Please let me know if you need more help/info or want me to do it on my end?

Thanks!

**Comment by superkhau** [comment]
This should definitely be backported.

**Pull request merged**